### DISCUSSION TITLE
VSIX availability date?

### DISCUSSION THREAD

**Discussion by AlchemyIntel** [microsoft/vscode-powerquery-sdk#119] [Q&A] [answered]
When will the VSCode SDK be available for public preview?

**Comment by mattmasson** [comment, marked as answer]
We are releasing incremental snapshots of the VSIX here - https://github.com/microsoft/vscode-powerquery-sdk/releases

We haven't set an official public preview date yet (where we'll publish to the VS Code marketplace). We're in an "early adopter" phase at this point. There are still a number of improvements we'd like to make before we're ready for a full public launch.

**Reply by AlchemyIntel** [reply]
Perfect. Thanks for the update!

**Comment by mattmasson** [comment]
To close out this item - the vscode-powerquery-sdk extension was published to the VS Code marketplace starting Oct 2022.
https://marketplace.visualstudio.com/items?itemName=PowerQuery.vscode-powerquery-sdk

We'll continue to publish the occasional interim release to this repo's [releases page](https://github.com/microsoft/vscode-powerquery-sdk/releases), but you can find the official release in the [VS Code Marketplace page](https://marketplace.visualstudio.com/items?itemName=PowerQuery.vscode-powerquery-sdk).
